packages/common/src/content.ts

```diff
@@ -45,3 +45,57 @@ export function getCategory(itemType: string = ""): string {
 export function getTypes(category: string = ""): string[] {
   return categories[category.toLowerCase()];
 }
+
+/**
+ * ```js
+ * import { getType } from "@esri/hub-common";
+ * //
+ * getType(item)
+ * > [ 'Hub Site Application' ]
+ * ```
+ * To do.
+ * @param layer Layer object.
+ * @param item Item object.
+ * @returns type of the input layer/item.
+ *
+ */
+export function getType(layer: any = {}, item: any = {}): string {
```

Comment:
Conceptually, do we have our own types? Or is it more like we use a superset of layer/item types? If it's the former, maybe a better name of this fn would be `getHubType()`?